features/README.md

```diff
@@ -483,7 +483,7 @@ foreach ($schema->keyspaces() as $keyspace) {
 
 ### Data Types
 
-The PHP driver for Apache Cassandra supports [a variety of datatypes](/features/core/datatypes/).
+The PHP driver for Apache Cassandra supports [a variety of datatypes](/features/datatypes/).
```

Comment:
The issue is the documentation has many, many links that don't work from GitHub markup, but work properly on doc.datastax.com. Doing this would make it the only exception. :) 

Here's a follow up story to address the bigger issue: https://datastax-oss.atlassian.net/browse/PHP-199